web/src/js/__tests__/ducks/_tflowServerAddressNull.js

```diff
@@ -0,0 +1,114 @@
+export default function(){
```

Comment:
This is a lot of code for what essentially is TFlow without a server address. I would propose we simply instantiate TFlow in our test function and manually unset the address attribute before we render the component?